src/js/components/Menu.js

```diff
@@ -39,6 +39,18 @@ var MenuDrop = React.createClass({
       down: this._onDownKeyPress
     };
     this.startListeningToKeyboard(this._keyboardHandlers);
+    var menuItems = this.refs.navContainer.getDOMNode().childNodes;
+    for (var i = 0; i < menuItems.length; i++) {
+      menuItems[i].setAttribute('role', 'menuitem');
```

Comment:
That makes sense! How do you think it'd be the best way to filter this? by tag? or by some other property?